examples/src/main/java/alluxio/examples/BasicOperations.java

```diff
@@ -68,7 +68,6 @@ public BasicOperations(AlluxioURI masterLocation, AlluxioURI filePath, ReadType
   public Boolean call() throws Exception {
     Configuration.set(PropertyKey.MASTER_HOSTNAME, mMasterLocation.getHost());
     Configuration.set(PropertyKey.MASTER_RPC_PORT, Integer.toString(mMasterLocation.getPort()));
-    ClientContext.init();
```

Comment:
should we remove ClientContext.init() in all places given that it is initialized in the static block of ClientContext class. Moreover, the init function doesn't seem to be threadsafe but ClientContext is annotated as threadsafe.